index.html

```diff
@@ -200,6 +200,9 @@ <h1 id="message-xso"><a href="#message-xso"><span>David</span> XSolidFigure</a><
       <h1 id="message-900"><a href="#message-900">9002</a></h1>
       <p>Seeing Aurora go is a shame, I remember joining Discord so I could talk on Aurora, I met some new friends there and all of the staff were friendly.<br>R.I.P Aurora Community, you will be dearly missed.</p>
 
+            <h1 id="message-jaf"><a href="#message-jaf"><span>Jaffa</span> Meme</a></h1>
```

Comment:
"Jaffa" should be your actual name (Dan), and "Meme" should be your username (JaffaMeme)